test/base.jl

```diff
@@ -77,6 +77,24 @@ end
   @test Array(Whatever{Int}.(CuArray([1]))) == Whatever{Int}.([1])
 end
 
+@testset "Cufunc" begin
+  import CUDAnative
```

Comment:
why is this needed?

Reply:
you are right, I forget that I change the implementation.